doc/rst/usingchapel/tasks.rst

```diff
@@ -139,7 +139,7 @@ available by means of the man(1) -m option, for example:
 
 .. code-block:: sh
```

Comment:
Since you're already updating this - on the line just above this example, could you please change "man(1) -m" to "man -M"?